tools/projmgr/docs/Manual/Overview.md

```diff
@@ -0,0 +1,1317 @@
+
+- `target-types` describe a different hardware target system and have therefore different API files for peripherals or a different hardware configuration.
+
+- `build-types` describe a build variant of the same hardware target. All `build-types` share the same API files for peripherals and the same hardware configuration, but may compile a different variant (i.e. with test I/O enabled) of an application.
+
+**Related Projects `iot-product.csolution.yml`**
+
+```yml
+target-types:
+  - type: Board
+    board: NUCLEO-L552ZE-Q
+
+  - type: Production-HW
+    device: STM32U5X          # specifies device
+      
+build-types:
+  - type: Debug
+    optimize: debug
+    debug: on
+
+  - type: Test
+    optimize: max
+    debug: on
+    
+solution:
+  - project: /security/TFM.cproject.yml
+    type: .Release
+  - project: /application/MQTT_AWS.cproject.yml
+  - project: /bootloader/Bootloader.cproject.yml
+    not-for-type: +Virtual
+```
+
+# Project Structure
+
+## Directory Structure  (old)
+
+This section describes how the files of `*.csolution.yml` should be organized to allow the scenarios described above:
+
+Source Directory                  | Content
+:-----------------------------------|:---------------
+`.`                                 | Contains one or more `*.csolution.yml` files that describes an overall application.
+`./<project>`                       | Each project has its own directory
+`./<project>/RTE+<target>`          | Configurable files that are specific to a target have a specific directory.
+`./<project>/RTE`                   | Configurable files that are common to all targets may have a common directory.
+`./<project>/Layer+<target>/<name>` | `*.clayer.yml` and related source files of a layers that are specific to a target have a specific directory.
+`./<project>/Layer/<name>`          | `*.clayer.yml` and related source files of a layers that are common to all targets may have a common directory.
+
+**Notes:**
+
+- `./<project>/RTE+<target>` contains the *.cprj file that is generated by `projmgr`
+- Directory names `RTE` and `Layer` should become configurable.  ToDo: analyze impact.
+
+The `./RTE` directory structure is maintained by tools and has the following structure. You should not modify the structure of this directory.
+
+`RTE[+<target>]` Directory        | Content
+:---------------------------------|:---------------
+`.../.<build-type>`               | Contains the file `RTE_Components.h` that is specific to a `build-type`.
+`.../<component class>`           | Configurable files for each component class are stored in sub-folders. The name of this sub-folder is derived from the component class name.
+`.../<component class>/<Dname>`   | Configurable files of the component class that are device specific. It is generated when a component has a condition with a `Dname` attribute. (strictly speaking no longer needed, backward compatiblity to MDK?)
```

Comment:
agree this does not sound useful here as the directory is already target specific